specification/2.0/README.md

```diff
@@ -3932,32 +3932,34 @@ Implementation of diffuse from [Lambert's Photometria](https://archive.org/detai
 
 Animations in glTF support spline interpolation with a cubic spline.
 
-The keyframes of a cubic spline in glTF have input and output values where each input value corresponds to three output values: in-tangent, spline vertex, and out-tangent.
+The keyframes of a cubic spline in glTF have input and output values where each input value corresponds to three output values of the same type: in-tangent, data tuple, and out-tangent.
```

Comment:
I'm not sure about using the term "data tuple" here. If the term spline vertex is confused with mesh vertex, then I would suggest spline point instead which looks like you use below.

Reply:
I was looking for a generic term which applies scalars, 3d vectors, and quaternions, I only used point below to explain the p in the formula. Perhaps data point ? In the future other types may be animated as well.